packages/admin-color-picker/src/core/HexInput.tsx

```diff
@@ -51,9 +50,7 @@ const HexInput: React.FC<IComponentProps & IVPAdminColorPickerProps> = ({ value,
                 )}
             </div>
         </div>
-        <ButtonBase classes={{ root: classes.clearButton }} onClick={() => onChange("")}>
-            <ClearIcon className={classes.clearIcon} fontSize="small" />
```

Comment:
you removed clearButton and clearIcon classes. Couldn't we keep them? Or don't we need them?

Reply:
I don't think they are necessary, because you can use `CometAdminClearInputButton`.  
But this would be an incompatible change...

Reply:
should we add a compatibility layer - or just ignore such a minor incompatible change?

Reply:
This PR is not very urgent, I think we could just merge into `next` instead of `master` if we want to be very strict about compatibility. But I don't think this specific feature is in use anywhere.